flood/src/main/java/com/dsvl/flood/Node.java

```diff
@@ -98,6 +111,34 @@ public boolean register() {
         return isRegistered;
     }
 
+    public boolean joinNetwork(List<Neighbour> existingPeers) {
```

Comment:
@KajanM said me, as a convention, not to use primitives, so instead of `boolean` use `Boolean` (other parts of code yet to be refactored..) just telling what he said.

Reply:
it is not applicable now, ignore it